src/utils/parseWorkflows.js

```diff
@@ -0,0 +1,10 @@
+export const parseWorkflows = workflows => {
+  return workflows.map(workflow => {
+    return {
+      ...workflow,
+      ui: {
+        originalContent: workflow
```

Comment:
Do you want to necessarily replace the entire `ui` object as a whole?
Or replace only the `originalContent` key in it? In which case you need:
```suggestion
        ...workflow.ui,
        originalContent: workflow
```

Reply:
`parseWorkflows` is used on fetching new data. We don't have previous workflows here to merge with the new workflows from the request.

Reply:
Okay :)